src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -164,9 +165,17 @@ public DynamicACLFilter(final UserProjectCache cache, final Settings settings, f
         this.esClient = TransportClient.builder().settings(clientSettings).addPlugin(SearchGuardSSLPlugin.class)
                 .addPlugin(SearchGuardPlugin.class) // needed for config update
                                                     // action only
-                .build().addTransportAddress(new InetSocketTransportAddress(new InetSocketAddress("localhost", 9300)));
+                .build();
+        try {
+            this.esClient.addTransportAddress(new InetSocketTransportAddress(new InetSocketAddress("localhost", 9300)));
+        }catch(Throwable e) {
```

Comment:
couldnt make the editor error go away... but finally found the expection in question. fixed.